scripts/update-mocks.mjs

```diff
@@ -0,0 +1,236 @@
+const astPathHandler = (path, context, baseAPI, tasks) => {
+  const dateComment = path.value.leadingComments?.find((comment) =>
+    comment.value.includes('Retrieved:')
+  );
+  const sourceComment = path.value.leadingComments?.find((comment) =>
+    comment.value.includes('Source:')
+  );
+
+  if (dateComment && sourceComment) {
+    const declaration =
+      // In case of variable declaration
+      path.value.declarations?.[0] ||
+      // In case of named export (variable declaration within named export)
+      path.value.declaration.declarations[0];
+
+    if (declaration.init.typeAnnotation) {
+      throw new Error(
+        'Do not cast the mock where it is defined, if you do have to cast it do it when exporting it'
+      );
+    }
+
+    // Will mutate the AST asynchronously, cannot wait to continue traversing
+    tasks.push(
+      // Async IIFE, returns a promise because async
+      (async () => {
+        const sourceURL = sourceComment.value.replace(/^ Source: /, '');
+        const parsedURL = new URL(
+          sourceURL,
+          sourceURL.startsWith('http') ? undefined : baseAPI
+        );
+
+        console.log(
+          '   - replacing content of variable',
+          chalk.bgBlueBright(declaration.id.name),
+          'with content from URL'
+        );
+        console.log('    ', chalk.bgBlueBright(parsedURL));
+
+        const response = await fetch(parsedURL, fetchOptions);
+        if (!response.ok) {
+          throw new Error(`Issue when downloading mock data from ${parsedURL}`);
+        }
+
+        const payload = await response.json();
+
+        // eslint-disable-next-line no-param-reassign
+        declaration.init = createValueExpression(payload);
+
+        // Update the date in the comment to keep track of when last updated
+        dateComment.value = dateComment.value.replace(
+          /\d{4}-\d{2}-\d{2}/,
+          new Date().toISOString().split('T')[0]
+        );
+      })()
+    );
+  }
+
+  context.traverse(path);
+};
+
+const processFile = async (file, baseAPI, dry) => {
+  const content = await fs.readFile(file, 'utf8');
+
+  const ast = recast.parse(content, recastOptions);
+
+  const tasks = [];
+
+  recast.visit(ast, {
+    visitExportNamedDeclaration(path) {
+      astPathHandler(path, this, baseAPI, tasks);
+    },
+    visitVariableDeclaration(path) {
+      astPathHandler(path, this, baseAPI, tasks);
+    },
+  });
+
+  await Promise.all(tasks);
+
+  let output = recast.print(ast).code;
+  // replace some ugly whitespaces between properties within objects
+  output = output.replaceAll(/,\n\s*\n/gm, ',\n');
+  // more formatting
+  output = prettier.format(output, {
+    ...prettierConfig,
+    parser: 'typescript',
+  });
+
+  if (!dry) {
+    console.log(`writing to ${file}`);
+    await fs.writeFile(file, output);
+  }
+
+  if (!tasks.length) {
+    console.log(
+      '  ',
+      chalk.bgYellow.italic('No mocks to update have been detected')
```

Comment:
![Screenshot 2024-06-12 at 14 24 07](https://github.com/ebi-uniprot/uniprot-website/assets/4357962/979b286e-ecb9-41b9-ac90-cd6b27997562)

Minor suggestion to change the color as it's a bit tricky read the grey text on the yellow background. 

```suggestion
      chalk.whiteBright.bgYellow.italic('No mocks to update have been detected')
```